src/applications/vaos/appointment-list/components/RequestedAppointmentDetailsPage.jsx

```diff
@@ -238,6 +238,7 @@ export default function RequestedAppointmentDetailsPage() {
       isCanceled,
       isCC,
       appointment,
+      cancelInfo,
```

Comment:
How did you figure this out??? I would not have thought that adding 'cancelInfo' to the dependency list would affect a tab stop.

Reply:
All I know is that after clicking `No, do not cancel` button, the page needs to focus on the h2 tag. This is the only useEffect that calls on the `scrollAndFocus` whenever the appointment object is present. Any change in the dependency list will trigger the focus on h2. The next step is to figure out which dependency triggers a change when the user clicks on `Cancel request` and `No, do not cancel` button. It turns out to be the `cancelInfo.appointmentToCancel`